src/diffusers/loaders.py

```diff
@@ -1101,7 +1183,7 @@ def _load_text_encoder_attn_procs(
         subfolder = kwargs.pop("subfolder", None)
         weight_name = kwargs.pop("weight_name", None)
         use_safetensors = kwargs.pop("use_safetensors", None)
-        network_alpha = kwargs.pop("network_alpha", None)
+        kwargs.pop("network_alpha", None)
```

Comment:
@takuma104 My understanding is that the returned attention processors from this class aren't used as instances at all, instead the parameters inside them are just pulled out and set on the weights. Is this understanding correct? and if so does it mean we can not set this network alpha value?

Reply:
nevermind, I see how it's used. can update